test/e2e/generate_kube_test.go

```diff
@@ -1025,6 +1025,17 @@ var _ = Describe("Podman kube generate", func() {
 		Expect(kube).Should(ExitCleanly())
 	})
 
+	It("multiple containers with differing user namespace configuration", func() {
+		name1 := "c1"
+		name2 := "c2"
+		_ = podmanTest.PodmanExitCleanly("run", "--userns", "auto:size=10", "-dt", "--name", name1, ALPINE, "top")
+		_ = podmanTest.PodmanExitCleanly("run", "--userns", "auto:size=10", "-dt", "--name", name2, ALPINE, "top")
```

Comment:
They have the exact same userns option so I would expect that to work and just use that one?! I agree that it should error if they are actually different.

Reply:
Fair enough, I'll check if the annotations are identical and allow if so